src/cli.rs

```diff
@@ -204,6 +204,22 @@ pub enum Commands {
         #[arg(long, value_enum, default_value_t = DEFAULT_LEVEL)]
         level: AaLevel,
     },
+
+    /// Delete genome(s) from a database (input: one id per line)
+    Delete {
+        /// Sketching database basename (so without .skm or .skd)
```

Comment:
Should be able to do this and ignore the suffix, like the other functions, so if you fix that change this to `Sketch database`